worldedit-bukkit/adapters/adapter-1_19/build.gradle.kts

```diff
@@ -12,6 +12,6 @@ repositories {
 }
 
 dependencies {
-    paperDevBundle("1.19.2-R0.1-20220805.230830-1")
+    paperDevBundle("1.19.2-R0.1-SNAPSHOT")
```

Comment:
Please pin the version, otherwise builds will fail, if more breaking changes are coming ahead mid-season, like this one, and nobody knows _where_ it's coming from.